R/sanity_checks.R

```diff
@@ -45,12 +45,12 @@ sanity_checks <- function(models,
     # statistic_vertical = FALSE: only one statistic can be displayed horizontally
     checkmate::assert_logical(statistic_vertical, len = 1, null.ok = FALSE)
     if (!statistic_vertical) {
-        if ((length(statistic_override) > 1) | (length(statistic) > 1)) {
+        if ((length(statistic_override)/length(models) > 1) | (length(statistic) > 1)) {
```

Comment:
space between operators

Reply:
Why did this line change?

Reply:
Without that change, I would not be able to pass one vector of (say) standard errors per model and show them next to the estimates.